website/docs/d/workspace_outputs.html.markdown

```diff
@@ -0,0 +1,93 @@
+---
+layout: "tfe"
+page_title: "Terraform Enterprise: tfe_workspace_outputs"
+sidebar_current: "docs-datasource-tfe-state-outputs"
+description: |-
+  Retrieves the State outputs per organization and workspace.
+---
+# Data Source: tfe_workspace_outputs
+
+This data source is used to retrieve the state outputs for a given workspace.
+It enables values in the outputs to be used dynamically in a terraform
+configuration.
+
+## Example Usage
+
+Using the `tfe_workspace_outputs` data source in a terraform configuration.
+
+In the example below, assume we have a state outputs that looks like this:
+
+```
+{
+  "version": <version>,
+  "terraform_version": "<terraform-version>",
+  ...
+  "outputs": {
+    "identifier": {
+      "value": "9023256633839603543",
+      "type": "string"
+    },
+    "records": {
+      "value": ["hashicorp.com", "terraform.io"],
+      "type": ["list", "string"]
+    },
+    "secret": {
+      "value": "token",
+      "type": "string",
+      "sensitive": true
+    }
+  },
+  "resources": [
+    ...
+  ]
+}
+```
+
+The `tfe_workspace_outputs` data source can now use `identifier` and `records`
+dynamically as seen below.
```

Comment:
* `can now` is temporal; this won't be new forever! 
* `dynamically` isn't of any consequence to the user, the fact that the type is variable is what's expected and sadly not the case until now 😆

```suggestion
Using the `tfe_workspace_outputs` data source, the outputs `identifier` and `records` can be used as seen below:
```